### PR TITLE
Null check on new OpenID Parameters setting

### PR DESCRIPTION
**Pull request by deanmarcussen**
Fixes https://github.com/OrchardCMS/OrchardCore/issues/9907

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void Configure(string name, OpenIdConnectOptions options)
                 }
             }
 
-            if (settings.Parameters.Any())
+            if (settings.Parameters?.Any() == true)
```
Comment (kevinchalet) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:101]: You need more C# 9 love in your life:

```suggestion
            if (settings.Parameters is { Length: > 0 })
```

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:101]: Would love to ;)

https://github.com/OrchardCMS/OrchardCore/blob/20d5554f78794d85acdb8b9e61a365f12b514b7b/src/OrchardCore.Build/OrchardCore.Commons.props#L9-L10

Comment (kevinchalet) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:101]: Weird, I've never had any issue with that combination 😄 
Well, then... 👍🏻

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:101]: Maybe we could just use ;)

    <PropertyGroup>
      <LangVersion>9.0</LangVersion>
      <RazorLangVersion>Latest</RazorLangVersion>
    </PropertyGroup>

Will do a PR this night if it works

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:101]: Worth a crack @jtkech, super sad

Where it fails (for me - Seb had it failing on his perf tests - and found somewhere in the docs that it isn't supported) is if you try and use a C# 9 feature in a razor page when building with core 3

**Pull request merged**